lib/uffizzi/cli/preview.rb

```diff
@@ -44,13 +44,21 @@ def describe(deployment)
       run(options, 'describe', nil, deployment)
     end
 
+    desc 'events', 'events'
+    def events(deployment)
+      return Cli::Common.show_manual(:events) if options[:help]
+
+      run(options, 'events', nil, deployment)
```

Comment:
file_path variable needs only for `handle_create_command` but it is nil in all other methods. Maybe will be better to use keyword arguments https://thoughtbot.com/blog/ruby-2-keyword-arguments?